e2e/testdata/credential-install-full.golden

```diff
@@ -0,0 +1,7 @@
+SECRET_ONE: foo
+/var/secret_two/data.txt
+bar
```

Comment:
Should we really print all the secrets on the stdout?

Reply:
This is coming from an invocation image whose sole purpose is to print out the secrets for testing purposes, it's not part of docker app (which would indeed be wrong to print such things)

Reply:
Oh right 😅